great_expectations/core/expectation_diagnostics/expectation_diagnostics.py

```diff
@@ -454,115 +436,6 @@ def _check_renderer_methods(
-                message=message,
-                passed=passed,
-                sub_messages=sub_messages,
-            )
-
-        impl = get_expectation_impl(camel_to_snake(expectation_camel_name))
-        try:
-            source_file_path = inspect.getfile(impl)
-        except TypeError:
-            sub_messages.append(
-                {
-                    "message": "inspect.getfile(impl) raised a TypeError (impl is a built-in class)",
-                    "passed": False,
-                }
-            )
-            return ExpectationDiagnosticCheckMessage(
-                message=message,
-                passed=passed,
-                sub_messages=sub_messages,
-            )
-
-        snaked_impl_name = camel_to_snake(impl.__name__)
-        source_file_base_no_ext = os.path.basename(source_file_path).rsplit(".", 1)[0]
-        with open(source_file_path) as fp:
-            code = fp.read()
-
-        if snaked_impl_name != source_file_base_no_ext:
-            sub_messages.append(
-                {
-                    "message": f"The snake_case of {impl.__name__} ({snaked_impl_name}) does not match filename part ({source_file_base_no_ext})",
-                    "passed": False,
-                }
-            )
-        else:
-            file_and_class_names_ok = True
-
-        if not BLACK_IMPORTED:
-            sub_messages.append(
-                {
-                    "message": "Could not find 'black', so cannot check linting",
-                    "passed": False,
-                }
-            )
-
-        if not ISORT_IMPORTED:
-            sub_messages.append(
-                {
-                    "message": "Could not find 'isort', so cannot check linting",
-                    "passed": False,
-                }
-            )
-
-        if BLACK_IMPORTED and ISORT_IMPORTED:
-            blacked_code = lint_code(code)
-            if code != blacked_code:
-                sub_messages.append(
-                    {
-                        "message": "Your code would be reformatted with black",
-                        "passed": False,
-                    }
-                )
-            else:
-                black_ok = True
-            isort_ok = isort.check_code(
-                code,
-                **isort.profiles.black,  # type: ignore[arg-type]
-                ignore_whitespace=True,
-                known_local_folder=["great_expectations"],
-            )
-            if not isort_ok:
-                sub_messages.append(
-                    {
-                        "message": "Your code would be reformatted with isort",
-                        "passed": False,
-                    }
-                )
-
-        passed = black_ok and isort_ok and file_and_class_names_ok
-        return ExpectationDiagnosticCheckMessage(
-            message=message,
-            passed=passed,
-            sub_messages=sub_messages,
-        )
-
```

Comment:
This breaks the diagnostics workflow, but fix is simple. Call to `ExpectationDiagnostics._check_linting(...)` needs to be removed from `great_expectations.expectations.expectation.py:2209`

Reply:
Done, thanks